tools/depends/configure.in

```diff
@@ -91,6 +91,25 @@ passed_ldflags="$LDFLAGS"
 passed_cxxflags="$CXXFLAGS"
 platform_cc=gcc
 platform_cxx=g++
+
+case $build in
+  arm*-*-linux-gnu*|arm*-*-linux-uclibc*)
+    build_os="linux"
+  ;;
+  *i686*-linux-gnu*|i*86*-*-linux-uclibc*)
+    build_os="linux"
+  ;;
+  x86_64*-linux-gnu*|x86_64-*-linux-uclibc*)
+    build_os="linux"
+  ;;
+  *darwin*)
+    build_os="osx"
+  ;;
+  *)
+    AC_MSG_ERROR(unsupported native build platform: $build)
```

Comment:
Nope and nope - error is here:

https://github.com/Memphiz/xbmc/blob/fixnative/m4/xbmc_arch.m4#L30

One ")" to much - i pulled the tools/depends/configure from the workspace and examined the line in question and realised it was in the xbmc_arch.m4 ... :)